### ISSUE TITLE
New router isn't consistently stripping uri?

### ISSUE THREAD

**Issue by kallqvist** [Mashape/kong#2017]
### Summary

I'm interrested in the new router in 0.10 so I'm testing the kong:0.10.0rc2 docker image.
I have a pretty basic setup with one api, one consumer and no plugins.

My issue is that only the first request is working.
All requests are going through to my upstream but it's only the first that will get the URI stripped.
If I restart Kong there will again be one request with stripped URI and all the following without.

### Steps To Reproduce

My config is:
```
{
data: [
{
http_if_terminated: true,
id: "be7f5aef-b921-48dd-ae65-57ca45f540fd",
retries: 5,
preserve_host: true,
created_at: 1485368263000,
upstream_url: "http://172.17.0.1:5050/",
https_only: false,
strip_uri: true,
name: "events",
uris: [
"/api/v1/events"
]
}
],
total: 1
}
```

And these are the logs from my upstream:
```
172.17.0.1 - - [25/Jan/2017 18:21:54] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [25/Jan/2017 18:21:58] "GET /api/v1/events HTTP/1.1" 404 -
172.17.0.1 - - [25/Jan/2017 18:22:20] "GET /api/v1/events HTTP/1.1" 404 -
172.17.0.1 - - [25/Jan/2017 18:22:20] "GET /api/v1/events HTTP/1.1" 404 -
```

**Comment by thibaultcha** [comment]
Hi, 

This was fixed by feeff486e94fdec96b19246341e72b813efdfcb9 and will be available in the next RC. 

Thanks for the report.

**Comment by kallqvist** [comment]
Great, thanks!
